relocations/quarkus-reactive-messaging-http-deployment/pom.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is it me or you haven't added these two modules in the relocations parent pom?

Reply:
yep, I missed it, adding